tensorboard/uploader/proto/export_service.proto

```diff
@@ -103,10 +106,18 @@ message StreamExperimentDataResponse {
   string run_name = 2;
   // The metadata of the tag `tag_name`.
   .tensorboard.SummaryMetadata tag_metadata = 3;
+
   // Data to store for the tag `tag_name.
-  ScalarPoints points = 4;
-  // Tensor data to store.
-  TensorPoints tensors = 5;
+  oneof data {
```

Comment:
I'm surprised that no other code needs to change as a result of this nesting.  There are no tests that access the existing scalar points subfield?

Reply:
I understand your point. `StreamExperimentDataResponse` has no usage in the OSS codebase, but tests of it do exists internally. Those tests will be run during sync.

Reply:
Ack. Thanks for the detailed explanation, @wchargin